examples/gno.land/p/demo/ownable/exts/authorizable/authorizable.gno

```diff
@@ -0,0 +1,70 @@
+// Package authorizable is an extension of p/demo/ownable;
+// It allows the user to instantiate an Authorizable struct, which extends
+// p/demo/ownable with a list of users that are authorized for something.
+// By using authorizable, you have a superuser (ownable), as well as another
+// authorization level, which can be used for adding moderators or similar to your realm.
+package authorizable
+
+import (
+	"std"
+
+	"gno.land/p/demo/ownable"
+)
+
+type Authorizable struct {
+	*ownable.Ownable
+	authorized map[std.Address]struct{}
+}
+
+func NewAuthorizable() *Authorizable {
+	a := &Authorizable{
+		ownable.NewOwnable(),
+		make(map[std.Address]struct{}),
+	}
+
+	// Add owner to auth list
+	a.authorized[a.Owner()] = struct{}{}
+	return a
+}
+
+func NewAuthorizableWithAddress(addr std.Address) *Authorizable {
+	a := &Authorizable{
+		ownable.NewOwnableWithAddress(addr),
+		make(map[std.Address]struct{}),
+	}
+
+	// Add owner to auth list
+	a.authorized[addr] = struct{}{}
+	return a
+}
+
+func (a *Authorizable) AddToAuthList(addr std.Address) error {
```

Comment:
missing the `DelFromAuthList`.

Reply:
I think it would be better like `AddAddress` or `AddMember`.

Reply:
I like the mention of AuthList in the function name; it's quite explicit and understandable